content/source/docs/cloud/registry/using.html.md

```diff
@@ -71,9 +63,12 @@ Use the "Examples" dropdown to navigate to example modules and use the  Readme/I
 
 ![Terraform Cloud screenshot: a module submodules detail page](./images/using-module-examples.png)
 
-## Using Private Modules in Terraform Configurations
+## Using Private Modules in Configurations
```

Comment:
@hashitop I changed the title to Using based on your comments earlier - to recap, this section is about how to define PMRs in your configuration, and we have another docs page that describes how to add PMRs/manage them in the registry: https://www.terraform.io/docs/cloud/registry/publish.html

Thoughts?